app/controllers/api/products_controller.rb

```diff
@@ -121,7 +121,7 @@ def product_query_includes
     end
 
     def paged_products_for_producers(producers)
-      Spree::Product.scoped.
+      Spree::Product.where(nil).
```

Comment:
I think that this works in Rails 3 as well.

Reply:
What about porting this one to master?

Reply:
Hi Maikel, I have asked this question on another PR or thread, Matt confirmed it is not compatible.

Reply:
Ah, okay. It seems to work in a simple case but there must be some edge cases that don't work. :+1: